src/index.html

```diff
@@ -3,11 +3,555 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+    rel="icon"
+    type="image/jpg"
+    href="images/favicon.jpg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://unpkg.com/aos@2.3.1/dist/aos.css">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <div class="page__body">
+      <div class="upward"></div>
+
+      <header class="header">
```

Comment:
Why is there a triple nesting here? One container per pair of children is more than enough:
```html
<header>
  <a><img></a>

  <h1>...</h1>
</header>
```

Reply:
so how can i fix it now?